code/src/components/Moviedetail.js

```diff
@@ -0,0 +1,194 @@
+import React, { useEffect, useState } from 'react'
+import { Link, useParams } from 'react-router-dom'
+import styled from 'styled-components'
+
+import { API_KEY } from 'Apis/Urls';
+
+const Moviedetail = () => {
```

Comment:
Very small thing, conventionally I would name it with camel case (but keeps the first letter as capital letter to differentiate between a react component and a plain javascript variables/functions). So in this case, I would use MovieDetail, as well as ErrorPage for consistency.